### DISCUSSION TITLE
Input in the manus.

### DISCUSSION THREAD

**Discussion by mercifr1** [insightsengineering/JointSurvivalModels.jl#26] [General]
Hi Yannik,
Naive question: How do I make suggestions on the manuscript in the .md file? Do I have to create a separate branch?
Thanks

**Comment by ya0** [comment]
You can add comments directly to the pull request #23  . Daniel already made a few maybe that can help you see how to make comments. This is quite a big pull request so many filea were changed, including the paper.md file.
